modules/core/src/main/java/org/apache/ignite/internal/pagemem/wal/record/WALRecord.java

```diff
@@ -291,7 +291,13 @@ public enum RecordType {
         CDC_DATA_RECORD(78, CUSTOM),
 
         /** Physical WAL record that represents a fragment of an entry update. */
-        DATA_PAGE_FRAGMENTED_UPDATE_RECORD(81, PHYSICAL);
+        DATA_PAGE_FRAGMENTED_UPDATE_RECORD(81, PHYSICAL),
+
+        // Custom GridGain WAL records should use indexes starting from 100.
+        // It allows avoiding overlapping with existing Apache Ignite records and not port placeholders to it.
```

Comment:
Well, on the one hand, you are right, it does not seem very reliable and requires adding the same comment into the Apache Ignite codebase. On the other hand, it looks ugly from the community point of view, that we contribute a placeholder without any functionality, it is just dead code that does not bring any value to the community and, potentially, it can be rejected, I think.
Anyway, it is not so important at this point. We can return to this topic when the feature is ready to be merged into the `master` branch.

Reply:
I agree with Denis, we should figure out what we do with new WAL records

Reply:
I almost sure that we will forget about this before merging to master :)
Maybe at least increase the gap